chargebee-js-vue/src/components/Element.vue

```diff
@@ -47,13 +47,14 @@ export default {
       if (cbComponent) {
         const options = this.fieldOptions;
         this.field = cbComponent.createField(this.id, options).at(`#${this.elementId}`)
-        if (this.$parent.onMount) this.$parent.onMount()
+        if (this.$parent.onMount) this.$parent.onMount();
```

Comment:
Ideally, This should be an emit event, and as a response we should be calling onMount method from the parent.

Reply:
Noted.
https://mychargebee.atlassian.net/browse/CHKOUTENGG-42575